resources/builder.inc.sh

```diff
@@ -698,24 +711,30 @@ _builder_add_chosen_action_target_dependencies() {
   while (( $i < ${#_builder_chosen_action_targets[@]} )); do
     action_target=${_builder_chosen_action_targets[$i]}
 
-    # If we have an internal dependency for the chosen action:target pair, add
-    # it to the list, but only if there is a defined output and that output is
-    # missing
+    # If we have an internal dependency for the chosen action:target pair
     if [[ ! -z ${_builder_internal_dep[$action_target]+x} ]]; then
       local dep_output=${_builder_internal_dep[$action_target]}
-      if [[ ! -z ${_builder_dep_path[$dep_output]+x} ]] &&
-          [[ ! -e "$KEYMAN_ROOT/${_builder_dep_path[$dep_output]}" ]]; then
-        if ! _builder_item_in_array "$dep_output" "${_builder_chosen_action_targets[@]}"; then
-          _builder_chosen_action_targets+=($dep_output)
-          new_actions+=($dep_output)
+      # If there is a defined output for this dependency
+      if [[ ! -z ${_builder_dep_path[$dep_output]+x} ]]; then
+        # If the output for the dependency is missing, or we have --force-deps
+        if [[ ! -e "$KEYMAN_ROOT/${_builder_dep_path[$dep_output]}" ]] || builder_is_full_dep_build; then
+          # Add the dependency to the chosen action:target list
+          if ! _builder_item_in_array "$dep_output" "${_builder_chosen_action_targets[@]}"; then
+            _builder_chosen_action_targets+=($dep_output)
+            new_actions+=($dep_output)
+          fi
         fi
       fi
     fi
     i=$((i + 1))
   done
 
   if [[ ${#new_actions[@]} -gt 0 ]]; then
-    echo "Automatically running following required actions with missing outputs:"
+    if builder_is_full_dep_build; then
+      echo "Automatically running all dependent actions due to --force-deps:"
```

Comment:
```suggestion
      echo "Automatically running all dependency actions due to --force-deps:"
```

Reply:
I think "dependency actions" is easier to understand than "dependent actions"